en/gps_compass/septentrio_asterx-rib.md

```diff
@@ -0,0 +1,170 @@
+# AsteRx OEM with Robotics Interface Board
+
+Septentrio is the leading worldwide supplier of OEM GPS / GNSS receivers. Our OEM receivers deliver accurate and

```

Comment:
This should IMO not refer to as 'our'.

Reply:
```suggestion
Septentrio is the leading worldwide supplier of OEM GPS / GNSS receivers. Septentrio OEM receivers deliver accurate and

```

Reply:
Flight log:
https://logs.px4.io/plot_app?log=ed164985-b9eb-4f50-80ec-55de2e83a2c5 

Do i need to test something specific for the flight log?

Reply:
@SeppeG I suspect @bkueng Wants to be able to see that the position uorb topics are being populated correctly and at the right rate. Further that the vehicle holds behaves as expected in a mode that relies on position.

Upshot, I would demonstrate a short mission and also a little manual flight in position mode.

Reply:
PS, yes, like that. I will commit your suggestion.

Reply:
> @SeppeG I suspect @bkueng Wants to be able to see that the position uorb topics are being populated correctly and at the right rate. Further that the vehicle holds behaves as expected in a mode that relies on position.

Upshot, I would demonstrate a short mission and also a little manual flight in position mode.

Exactly, just that we're sure it's working as expected.